docs/external-postgres.md

```diff
@@ -0,0 +1,59 @@
+# Using an external database with the Chronicle Helm chart
+
+Chronicle uses a PostgreSQL database to store data. By default, Chronicle will
+use an embedded PostgreSQL database. However, you can configure Chronicle to
+use an external PostgreSQL database instead.
+
+## Configure Chronicle to use an external PostgreSQL database
+
+The default values.yaml for the Chronicle helm chart configures an
+internal Postgres database.
```

Comment:
```suggestion
internal PostgreSQL database.
```
We switch back and forth somewhat, probably best to make this and the one below the same as the others.

Reply:
Thanks I've updated to match PostgreSQL everywhere